core/domain/suggestion_services.py

```diff
@@ -166,6 +166,22 @@ def _update_suggestion(suggestion):
     suggestion_model.put()
 
 
+def update_skill_id_for_question_suggestion(suggestion, skill_id):
```

Comment:
See above; hopefully we can remove this.

Also if you keep this function, it needs tests.

Reply:
I removed the function